dataplicity/client.py

```diff
@@ -81,13 +81,7 @@ def _init(self):
             log.info("serial=%s", self.serial)
             log.info("poll=%s", self.poll_rate_seconds)
 
-            self.directory_scanner = DirectoryScanner(
-                self.exit_event,
-                self.remote_directory_path,
-                self.remote,
-                self.serial,
-                self.auth_token,
-            )
+            self.directory_scanner = DirectoryScanner(self.remote_directory_path)
```

Comment:
This object has become much simpler. Agent no longer performs scan at regular interviews. The scans can be requested by Router, so we can change the schedule as requried.